common/djangoapps/philu_commands/management/commands/notify_non_active_users.py

```diff
@@ -0,0 +1,76 @@
+from common.lib.mandrill_client.client import MandrillClient
+from courseware.models import StudentModule
+from openedx.core.djangoapps.content.course_overviews.models import CourseOverview
+from openedx.core.djangoapps.timed_notification.core import get_course_link
+from openedx.features.course_card.helpers import get_course_open_date
+from student.models import CourseEnrollment
+from django.core.management.base import BaseCommand
+from openedx.features.course_card.models import CourseCard
+from datetime import datetime, timedelta
+
+from logging import getLogger
+log = getLogger(__name__)
+
+
+class Command(BaseCommand):
+    help = """
+    Send Notifications prompts to users who have not entered into course after enrollment.
```

Comment:
Lets comment the case here where the user enrolls 7 days after the course started and doesnt enter the course ever since.